testsuite/galleon/pom.xml

```diff
@@ -22,7 +22,7 @@
     <artifactId>wildfly-ts-galleon-parent</artifactId>
     <packaging>pom</packaging>
     <name>WildFly Test Suite: Galleon tests (parent)</name>
-    <modules>
-        <module>update</module>
-    </modules>
+            <modules>
```

Comment:
Not sure that this formatting is required.

Reply:
Thanks @jfdenise -- that's the result of my taking an approach to something and then abandoning it incompletely. I'll correct that and merge this.